src/main/java/org/mule/extension/sftp/internal/operation/SftpOperations.java

```diff
@@ -158,7 +160,10 @@ public void write(@Connection SftpFileSystemConnection fileSystem,
                     @Content @Summary("Content to be written into the file") InputStream content,
                     @Optional(defaultValue = "true") boolean createParentDirectories,
                     @Optional(defaultValue = "false") boolean lock, @Optional(
-                        defaultValue = "OVERWRITE") @Summary("How the file is going to be written") @DisplayName("Write Mode") FileWriteMode mode) {
+                        defaultValue = "OVERWRITE") @Summary("How the file is going to be written") @DisplayName("Write Mode") FileWriteMode mode,
+                    @Placement(tab = ADVANCED_TAB) @Optional(defaultValue = "STANDARD") WriteStrategy writeStrategy,
+                    @Placement(tab = ADVANCED_TAB) @Optional(
+                        defaultValue = "BUFFER_SIZE_1KB") CustomWriteBufferSize bufferSizeForWriteStrategy) {
```

Comment:
Please make the default same as the one in the existing implementation (8K)

Reply:
Done